src/test/a11y/tests/media-routes.test.ts

```diff
@@ -122,5 +122,39 @@ describe('Accessibility - Media User Routes', () => {
                 testAccessibility(url, '', true, { 'search-input': 'Invalid case number' });
             });
         });
+
+        describe('Bulk Unsubscribe Confirmation Page', () => {
+            const url = '/bulk-unsubscribe-confirmation';
+
+            describe('with no input data', () => {
+                testAccessibility(url, '', true, { 'input-autocomplete': '' });
```

Comment:
'input-autocomplete' is not the right post body for this test. Please check the controller code